_episodes/01-run-quit.md

```diff
@@ -5,55 +5,153 @@ exercises: 0
 questions:
 - "How can I run Python programs?"
 objectives:
-- "Launch the Jupyter Notebook, create new notebooks, and exit the Notebook."
+- "Launch the JupyterLab server." 
+- "Create a new Python script." 
+- "Create a Jupyter notebook."
+- "Shutdown the JupyterLab server."
+- "Understand the difference between a Python script and a Jupyter notebook."
 - "Create Markdown cells in a notebook."
 - "Create and run Python cells in a notebook."
 keypoints:
-- "Python programs are plain text files."
+- "Python scripts are plain text files."
 - "Use the Jupyter Notebook for editing and running Python."
 - "The Notebook has Command and Edit modes."
 - "Use the keyboard and mouse to select and edit cells."
 - "The Notebook will turn Markdown into pretty-printed documentation."
 - "Markdown does most of what HTML does."
 ---
-## Python programs are plain text files.
 
-*   They have the `.py` extension to let everyone (including the operating system) 
```

Comment:
Other maintainer's thoughts on moving the various `Python programs are plain text files` bullet points down to separate sections below? It might still be worthwhile to repeat them up here and below, something like:

> The `.py` extension is a convention not a requirement and it is common to edit Python scripts via an IDE or text editor. However, we will be using Jupyter Lab and Jupyter Notebooks to modify them as Jupyter offers a number of helpful features.